### DISCUSSION TITLE
Al crear documento, regresa un error de usoCfdi y regimenFiscal vacíos cuando sí se envían en el Request

### DISCUSSION THREAD

**Discussion by JuliusLancelot** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#64] [Q&A] [answered]
Este es el _model_ en el _request_:

```
"""model"": {
  ""documento"": {
    ""concepto"": {
      ""codigo"": ""5""
    },
    **""serie"": ""C"",
    ""folio"": ""657450"",**
    ""fecha"": ""2023-12-05T12:49:46"",
    ""cliente"": {
      ""codigo"": ""112204"",
      ""razonSocial"": ""PUBLICO EN GENERAL"",
      ""rfc"": ""XAXX010101000"",
      ""tipo"": ""Cliente"",
      ""usoCfdi"": ""S01"",
      ""regimenFiscal"": ""616"",
    },
    ""moneda"": {
      ""id"": ""1"",
      ""nombre"": ""Peso Mexicano""
    },
    ""tipoCambio"": ""1"",
    ""agente"": {
      ""id"": ""3"",
      ""codigo"": ""126""
    },
    ""formaPago"": ""01"",
    ""metodoPago"": ""PUE"",
    ""movimientos"": [
      {
        ""producto"": {
          ""codigo"": ""04901"",
          ""nombre"": ""2365816"",
          ""tipo"": ""Producto"",
          ""claveSat"": ""31201600""
        },
        ""almacen"": {
          ""codigo"": ""M01"",
          ""nombre"": ""M01""
        },
        ""unidades"": ""2.000000"",
        ""precio"": ""176.49"",
        ""subtotal"": 352.98000000,
        ""descuentos"": {
          ""descuento1"": {
            ""tasa"": ""0.00"",
            ""importe"": ""0.000000""
          }
        },
        ""impuestos"": {
          ""impuesto1"": {
            ""tasa"": ""8.000"",
            ""importe"": ""28.240000""
          }
        },
        ""total"": 381.22000000
      }
    ],
    ""direccionFiscal"": {
      ""calle"": ""xxx"",
      ""numeroExterior"": ""x"",
      ""colonia"": ""x"",
      ""ciudad"": ""x"",
      ""estado"": ""x"",
      ""codigoPostal"": ""x"",
      ""pais"": ""x""
    }
  }
}"
```
**Este es el error:**

`"""errorMessage"": ""Mensaje:[|El atributo 'RegimenFiscalReceptor' no es válido. El valor '' no es válido según su tipo de datos 'http://www.sat.gob.mx/sitio_internet/cfd/catalogos:c_RegimenFiscal' - Error en la restricción de Enumeration.|El atributo 'UsoCFDI' no es válido. El valor '' no es válido según su tipo de datos 'http://www.sat.gob.mx/sitio_internet/cfd/catalogos:c_UsoCFDI' - Error en la restricción de Enumeration.\nArchivo: C:\\Compac\\Empresas\\xxx\\**_FacturaalContadoC657360.xml**] Numero:[130101]"""`

**Comment by JuliusLancelot** [comment]
Estoy notando que el XML en el error dice _FacturaalContadoC657360.xml y es el xml de otro documento. Por qué estará generando eso?

**Reply by JuliusLancelot** [reply, marked as answer]
Ya encontré el error. Es cuando el documento se crea en un momento en el que no se había enviado dicha información en el cliente, y aunque se actualice la información en el cliente, al intentar timbrarlo ya no lo toma del cliente nuevamente sino que lo toma de los datos que se quedaron en el documento. La solución podría ser un actualizar Documento o borrar el documento.
Como actualizar requiere conocer campos de base de datos, elegí borrarlo y volverlo a crear.

Saludos!

**Reply by AndresRamos** [reply]
Mi recomendacion es que cuando el documento no se cree correctamente, que este se elimine y se vuelva a crear.
